yaml/pois.yaml

```diff
@@ -400,7 +395,7 @@ filters:
   ############################################################
   # aerodrome
   - filter:
-      aeroway: [aerodrome, airport]
+      aeroway: [aerodrome, airport, heliport]
```

Comment:
There are some very large heliports in Hawaii so maybe heliport should be 14, but I'm letting this slide...